test/integration/create-operator-in-operator/00-dream.yaml

```diff
@@ -25,14 +25,14 @@ spec:
       metadata:
         name: inner-instance
         labels:
-          kudo.dev/operator: test-operator
+          kudo.dev/operator: configmap-operator
       spec:
         operatorVersion:
           kind: OperatorVersion
-          name: test-operator-1.0
+          name: configmap-operator-1.0
```

Comment:
this test was actually lying to us all the time, because we were passing `existing` to health here https://github.com/kudobuilder/kudo/pull/1187/files#diff-baf64983976d84fa08ff5a16d1474e7cL78 the health module got an object with Generation=0 and ObservedGeneration was also 0 because there are no controllers running in integration test so it was immediately marked as healthy. With the fix to the health module, this was no longer passing so I had to choose an operator that a bit bypasses the health module